doc/devel/dependencies.rst

```diff
@@ -1,15 +1,15 @@
 .. _dependencies:
 
-============
+************
 Dependencies
-============
+************
 
-Runtime dependencies
-====================
+Runtime
```

Comment:
I like the longer versions better because they are more self-contained.
Whenever I read these single-adjective headings, I feel they are incomplete and I should mentally add “dependencies “. Headings should describe the topic of the section. A single adjective cannot do that, because adjectives only qualify things, but that thing is not stated.

Reply:
Self-contained subheadings also work better if you want to link a specific subheading from some other part of the docs I think.

Reply:
I don't feel very strongly, but personally prefer the proposed concise headings.  I think that if it says 

- Dependencies
  - Runtime
  - Testing
  - Documentation
  - etc

the "dependencies" is understood, and is better than 

- Dependencies
  - Runtime dependencies
  - Testing dependencies
  - Documentation dependencies
  - etc dependencies

I appreciate the cross linking point, but we can always inject the proper description``` :ref:`runtime dependencies <runtime_dependencies>` ```